src/main/java/org/fundacionjala/coding/carlos/countfrequentitem/CountFrequentItem.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.carlos.countfrequentitem;
+
+public class CountFrequentItem {
+
+    /**
+     * @param arrayNumbers to find the most frequent item
+     * @return the number of times that the most frequent item is repeated in an array.
+     */
+    public int getMostFrequent(final int[] arrayNumbers) {
+        int number = 0;
+        int repeated = 0;
+        for (int arrayNumber : arrayNumbers) {
```

Comment:
Please consider removing duplicates from the original array first, according to your solution every number in the array will compute `verifyFrequency()` method including numbers already processed (the duplicated ones). Consider removing duplicates to improve performance with:
```java
LinkedHashSet<Integer> linkedHashSet = new LinkedHashSet<>(Arrays.asList(arrayNumbers));
Integer[] numbersWithoutDuplicates = linkedHashSet.toArray(new Integer[] {});
```